docs/getting-started/install-configure.md

```diff
@@ -263,17 +265,67 @@ kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/master/
 ```
 
 </div>
-<div class="tab-pane fade" id="provider-tab-gcp" markdown="1">
+<div class="tab-pane fade" id="aws-new-tab-1" markdown="1">
 
-### Install GCP Provider
+### Install Configuration Package
 
 ```console
-kubectl crossplane install provider crossplane/provider-gcp:v0.14.0
+kubectl crossplane install configuration registry.upbound.io/xp/getting-started-with-aws-with-vpc:latest
 ```
 
-Wait until the provider becomes healthy:
+Wait until all packages become healthy:
+```
+kubectl get pkg --watch
 ```
-kubectl get provider.pkg --watch
+
+### Get AWS Account Keyfile
+
+Using an AWS account with permissions to manage RDS databases:
+
+```console
+AWS_PROFILE=default && echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $AWS_PROFILE)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $AWS_PROFILE)" > creds.conf
+```
+
+### Create a Provider Secret
+
+```console
+kubectl create secret generic aws-creds -n crossplane-system --from-file=key=./creds.conf
+```
+
+### Configure the Provider
+
+We will create the following `ProviderConfig` object to configure credentials
+for AWS Provider:
```

Comment:
I considered leaving out all of the `ProviderConfig` steps and putting them into the configuration packages themselves so that users would only need to create the provider secret. This would make the guide slightly more streamlined, but I opted against doing this for now since it would hide an important concept. Definitely open to more thoughts on this though.

Reply:
I think that's a good call.